main.rb

```diff
@@ -0,0 +1,143 @@
+require_relative 'app'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  def create_a_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    puts 'Person created successfully '
+    puts
+    @app.create_a_teacher(name, specialization, age)
+  end
```

Comment:
- [ ] Great job working on this milestone. Your main.rb should be just as the entry point into the app as per requiremnents. All your methods should be defined in app.rb. Therefore in your main.rb, you should have something similar to:

```
def main
  app = App.new()

  response = nil

  # *Present the user with a list of options to perform.
  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by entering a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
    when "4"
      app.create_book()
    when "5"
      app.create_rental()
    when "6"
      app.list_rentals_for_person_id()
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()

```